frontend/src/views/GAccount.vue

```diff
@@ -270,14 +273,23 @@ export default {
     GAccountAvatar,
   },
   inject: ['api', 'logger', 'yaml'],
+  setup () {
+    const configStore = useConfigStore()
+    const grantTypes = configStore.grantTypes
+
+    const grantType = ref(head(grantTypes))
+
+    return {
+      grantType,
+      grantTypes,
+    }
+  },
```

Comment:
Use `mapState` instead of setup. 
```
  computed: {
    ...
    ...mapState(useKubeconfigStore, [
      'kubeconfigData',
      'isKubeconfigEnabled',
    ]),
    ...mapState(useConfigStore, [
      'grantTypes',
    ]),
    ... 
},
```

Reply:
✅